docker/Dockerfile

```diff
@@ -18,20 +18,30 @@
 FROM openjdk:8-jre
 
 # Unomi environment variables
-ENV KARAF_INSTALL_PATH /opt
-ENV KARAF_HOME $KARAF_INSTALL_PATH/apache-unomi
-ENV PATH $PATH:$KARAF_HOME/bin
+ENV UNOMI_HOME /opt/apache-unomi
+ENV PATH $PATH:$UNOMI_HOME/bin
+
 ENV KARAF_OPTS "-Dunomi.autoStart=true"
-WORKDIR $KARAF_HOME
 
-RUN wget http://apache.mirrors.pair.com/incubator/unomi/1.3.0-incubating/unomi-1.3.0-incubating-bin.tar.gz
-RUN tar -xzf unomi-1.3.0-incubating-bin.tar.gz
-RUN mv unomi-1.3.0-incubating/* .
-RUN rm unomi-1.3.0-incubating-bin.tar.gz
-RUN rm -r unomi-1.3.0-incubating
+ENV ELASTICSEARCH_HOST localhost
+ENV ELASTICSEARCH_PORT 9300
+
+ENV UNOMI_VERION "1.4.0"
```

Comment:
I just noticed there's a small typo in VERION. Should be VERSION. Can you correct that ?